src/PJV.test.ts

```diff
@@ -31,15 +31,15 @@ const npmWarningFields = {
 
 describe("Basic", () => {
 	test("Input types", () => {
-		assert.ok(PJV.validate("string").critical, "string");
-		assert.ok(PJV.validate("{").critical, "malformed object");
-		assert.ok(PJV.validate("[]").critical, "array");
-		assert.ok(PJV.validate('"malformed"').critical, "quoted string");
-		assert.ok(PJV.validate("42").critical, "number");
-		assert.ok(PJV.validate("null").critical, "null");
-		assert.ok(PJV.validate("true").critical, "true");
-		assert.ok(PJV.validate("false").critical, "false");
-		assert.ok(PJV.validate({} as string).critical, "literal object");
+		assert.ok(validate("string").critical, "string");
+		assert.ok(validate("{").critical, "malformed object");
+		assert.ok(validate("[]").critical, "array");
+		assert.ok(validate('"malformed"').critical, "quoted string");
+		assert.ok(validate("42").critical, "number");
+		assert.ok(validate("null").critical, "null");
+		assert.ok(validate("true").critical, "true");
+		assert.ok(validate("false").critical, "false");
+		assert.ok(validate({} as string).critical, "literal object");
 	});
 });
 describe("NPM", () => {
```

Comment:
[Testing] WDYT about adding a test to make sure `PJV.validate` _is_ `validate`? Think that's overkill?

```suggestion
});

describe("PJV", () => {
	test("properties", () => {
		assert.strictEqual(PJV.validate, validate);
	});
});

describe("NPM", () => {
```

Reply:
Never hurts to add more tests.  Updated